### DISCUSSION TITLE
不是哥们，真有绳网啊

### DISCUSSION THREAD

**Discussion by YOLOjj** [share121/inter-knot#4673] [灌水]
[今天刷视频刷到的，过来一看我咧个绳网啊]
来都来了，随便放张图吧
![屏幕截图 2024-07-18 013541](https://github.com/user-attachments/assets/6bf35ac0-3dfe-4fa9-817c-1432cc92add2)
大佬NB
灌水灌水灌水灌水灌水灌水灌水灌水灌水灌水灌水灌水灌水灌水灌水灌水。。。

**Comment by YOLOjj** [comment]
诶，我图呢

**Reply by Dave-12138** [reply]
你看到uploading就说明图还在上传，这时候图还没传完点提交自然就没图了<img src="https://bbs-static.miyoushe.com/static/2024/06/19/f7d6b0099c022d557d12c6662b132406_7125757908673767353.png" alt="流萤 也挺好" width="75" >

**Reply by YOLOjj** [reply]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
哦哦好嘟

**Comment by mztxy229** [comment]
哈哈哈哈哈哈哈哈哈哈哈好傻的楼主（

**Reply by YOLOjj** [reply]
什么话，什么话这是

**Comment by Power-Bai** [comment]
欢迎来到绳网
![1724016804056](https://github.com/user-attachments/assets/e293f940-056f-424b-9c46-6b82f396d940)

**Reply by YOLOjj** [reply]
好图，偷
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >
